src/main/java/com/crumbs/fss/controller/MainController.java

```diff
@@ -41,7 +41,7 @@
         Page<Restaurant> restaurants = restaurantSearchService.getRestaurants(pageRequest)
                 .orElseThrow();
         if (filter != null && filter.length > 0)
-            restaurants = restaurantSearchService.filterResults(filter, pageRequest);
+            restaurants = restaurantSearchService.filterRestaurantResults(null, filter, pageRequest);

```

Comment:
Passing null as an argument, is this okay?

Reply:
I think this is ok